app/delegate-server/HydraNodeApi/Types/Commit.purs

```diff
@@ -0,0 +1,174 @@
+module DelegateServer.HydraNodeApi.Types.Commit
```

Comment:
Generally I am not fond of calling packages Types (if we are not working on a compiler of course). I believe it's always easier to put types along with anything else and call the module with a name that comes from the domain area something that is known as screaming architecture (this is not a call for changes).